packages/arduino_ide.rb

```diff
@@ -0,0 +1,66 @@
+require 'package'
```

Comment:
This wouldn't work?
```
ENV["CREW_NOT_STRIP"] = 1
require 'package'
```

Reply:
Tried that.

Reply:
How about this?
```
ENV["CREW_NOT_STRIP"] = 1
require 'const'
require 'package'
```

Reply:
Actually, that won't work since const is not a class.

Reply:
This might work:
```
ENV["CREW_NOT_STRIP"] = 1
load 'const.rb'
require 'package'
```

Reply:
This is not going to work either.  I have a solution.  Change crew:
```
diff --git a/crew b/crew
index 91fb2614..cf947aa1 100755
--- a/crew
+++ b/crew
@@ -523,7 +523,7 @@ end
 
 def compress_doc (dir)
   # check whether crew should compress
-  return if CREW_NOT_COMPRESS || !File.exist?("#{CREW_PREFIX}/bin/compressdoc")
+  return if CREW_NOT_COMPRESS || ENV['CREW_NOT_COMPRESS'] || !File.exist?("#{CREW_PREFIX}/bin/compressdoc")
 
   if Dir.exist? dir
     system "find #{dir} -type f ! -perm -200 | xargs -r chmod u+w"
@@ -564,7 +564,7 @@ end
 
 def strip_find_files (find_cmd, strip_option = "")
   # check whether crew should strip
-  return if CREW_NOT_STRIP || !File.exist?("#{CREW_PREFIX}/bin/strip")
+  return if CREW_NOT_STRIP || ENV['CREW_NOT_STRIP'] || !File.exist?("#{CREW_PREFIX}/bin/strip")
```
This will eliminate the need to redefine constants which we shouldn't be doing anyway.  Constants are just that and are not meant to be changed.

Reply:
Bingo. Updated new PR and testing this one for confirmation.